db/migrate/20150910010112_create_team_memberships.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTeamMemberships < ActiveRecord::Migration
+  def change
+    create_table :team_memberships do |t|
+      t.belongs_to :team
+      t.belongs_to :user
+      t.integer :role, default: 0
```

Comment:
Should we change this to be more descriptive if we're gonna use a boolean? E.g., `is_admin`?